physics/module_MYNNSFC_wrapper.F90

```diff
@@ -19,14 +19,19 @@ end subroutine mynnsfc_wrapper_finalize
 
 !>\defgroup gsd_mynn_sfc GSD MYNN Surface Layer Scheme Module
 !> \brief This scheme (1) performs pre-mynnsfc work, (2) runs the mynn sfc layer scheme, and (3) performs post-mynnsfc work
+#if 0
```

Comment:
This #if 0 and the #endif in line 26 are not required

Reply:
I took them out.